content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -1179,10 +1179,16 @@ The process between Alice and Bob verifying each other would be:
 
 ###### QR code format
 
-The QR codes to be displayed and scanned using this format will encode binary
-strings in the general form:
+The QR codes to be displayed and scanned must be
+[ISO/IEC 18004:2015](https://www.iso.org/standard/62021.html) compatible and
```

Comment:
[sadly yes, the spec for QR codes is copyrighted; this seems to be the best we can do.]